testsuite/features/build_validation/init_clients/monitoring_server.feature

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@monitoring_server
+Feature: Bootstrap the Monitoring Server
+
+  Scenario: Clean up sumaform leftovers on the Monitoring Server
+    When I perform a full salt minion cleanup on "monitoring_server"
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Bootstrap the Monitoring Server
+    When I follow the left menu "Systems > Bootstrapping"
+    Then I should see a "Bootstrap Minions" text
+    When I enter the hostname of "monitoring_server" as "hostname"
+    And I enter "22" as "port"
+    And I enter "root" as "user"
+    And I enter "linux" as "password"
+    And I select "1-monitoring_server_key" from "activationKeys"
+    And I select the hostname of "proxy" from "proxies" if present
+    And I click on "Bootstrap"
+    And I wait until I see "Successfully bootstrapped host!" text
+    And I wait until onboarding is completed for "monitoring_server"
+
+  Scenario: Check the new bootstrapped Monitoring Server in System Overview page
+    When I follow the left menu "Salt > Keys"
+    Then I should see a "accepted" text
+    And the Salt master can reach "monitoring_server"
+
+@proxy
+  Scenario: Check connection from Monitoring Server to proxy
+    Given I am on the Systems overview page of this "monitoring_server"
+    When I follow "Details" in the content area
+    And I follow "Connection" in the content area
+    Then I should see "proxy" short hostname
+
+@proxy
+  Scenario: Check registration on proxy of Monitoring Server
+    Given I am on the Systems overview page of this "proxy"
+    When I follow "Details" in the content area
+    And I follow "Proxy" in the content area
+    Then I should see "monitoring_server" hostname
+
+  Scenario: Check events history for failures on Monitoring Server
+    Given I am on the Systems overview page of this "monitoring_server"
+    Then I check for failed events on history event page
```

Comment:
Do you miss here the scenarios to apply the prometheus and grafana salt formulas to our monitoring server. And after that, check that prometheus service is up and running and same for grafana.

Reply:
That's being done already, in the smoke tests - this is just to "start" the monitoring server minion and make sure it's working normally